rbac/app/config.py

```diff
@@ -43,24 +37,24 @@ def getenv(name, default):
     "AES_KEY": "1111111111111111111111111111111111111111111111111111111111111111",
 }
 
-SERVER_HOST = getenv("SERVER_HOST", DEFAULT_CONFIG["SERVER_HOST"])
-SERVER_PORT = getenv("SERVER_PORT", DEFAULT_CONFIG["SERVER_PORT"])
-VALIDATOR_HOST = getenv("VALIDATOR_HOST", DEFAULT_CONFIG["VALIDATOR_HOST"])
-VALIDATOR_PORT = getenv("VALIDATOR_PORT", DEFAULT_CONFIG["VALIDATOR_PORT"])
+SERVER_HOST = os.getenv("SERVER_HOST", DEFAULT_CONFIG["SERVER_HOST"])
+SERVER_PORT = os.getenv("SERVER_PORT", DEFAULT_CONFIG["SERVER_PORT"])
+VALIDATOR_HOST = os.getenv("VALIDATOR_HOST", DEFAULT_CONFIG["VALIDATOR_HOST"])
+VALIDATOR_PORT = os.getenv("VALIDATOR_PORT", DEFAULT_CONFIG["VALIDATOR_PORT"])
 VALIDATOR_TIMEOUT = int(
-    getenv("VALIDATOR_TIMEOUT", DEFAULT_CONFIG["VALIDATOR_TIMEOUT"])
+    os.getenv("VALIDATOR_TIMEOUT", DEFAULT_CONFIG["VALIDATOR_TIMEOUT"])
 )
-VALIDATOR_REST_HOST = getenv(
+VALIDATOR_REST_HOST = os.getenv(
     "VALIDATOR_REST_HOST", DEFAULT_CONFIG["VALIDATOR_REST_HOST"]
 )
-VALIDATOR_REST_PORT = getenv(
+VALIDATOR_REST_PORT = os.getenv(
     "VALIDATOR_REST_PORT", DEFAULT_CONFIG["VALIDATOR_REST_PORT"]
 )
-DB_HOST = getenv("DB_HOST", DEFAULT_CONFIG["DB_HOST"])
-DB_PORT = getenv("DB_PORT", DEFAULT_CONFIG["DB_PORT"])
-DB_NAME = getenv("DB_NAME", DEFAULT_CONFIG["DB_NAME"])
-AES_KEY = getenv("AES_KEY", DEFAULT_CONFIG["AES_KEY"])
-SECRET_KEY = getenv("SECRET_KEY", DEFAULT_CONFIG["SECRET_KEY"])
+DB_HOST = os.getenv("DB_HOST", DEFAULT_CONFIG["DB_HOST"])
+DB_PORT = os.getenv("DB_PORT", DEFAULT_CONFIG["DB_PORT"])
+DB_NAME = os.getenv("DB_NAME", DEFAULT_CONFIG["DB_NAME"])
+AES_KEY = os.getenv("AES_KEY", DEFAULT_CONFIG["AES_KEY"])
```

Comment:
This fails to get the default AES key because the environment has a blank string by default.

Reply:
Where does an environment variable named "AES_KEY" occur by default? In the OS? From NEXT? 

I am wondering because I just opened a terminal, did a 'printenv', and I don't have an 'AES_KEY' environment variable on my system.

Reply:
You would ``printenv``` in the server docker container instance.

docker-compose.yaml
    environment:
      - SECRET_KEY=${SECRET_KEY}
      - AES_KEY=${AES_KEY}

Reply:
This PR should have failed Travis if not for the lack of an e2e API test.